build-system/tasks/helpers.js

```diff
@@ -38,7 +39,7 @@ const {jsBundles} = require('../compile/bundles.config');
 const {log, logLocalDev} = require('../common/logging');
 const {removeFromClosureBabelCache} = require('../compile/pre-closure-babel');
 const {thirdPartyFrames} = require('../test-configs/config');
-const {watch: fileWatch} = require('chokidar');
+const {watch: fileWatch, watch} = require('chokidar');
```

Comment:
Duplicate? Maybe get rid of `fileWatch`?

Reply:
Fixed!